### PR TITLE
Print SSL errors for assistance with debug.

### PR DESCRIPTION
**Pull request by psavery**

### CODE REVIEW

**Comment by psavery** [comment]
This commit is just for print SSL errors. This can assist with SSL problems in the future. I can confirm that this builds on both MSVC 2010 and Ubuntu 14.04 successfully.

I'll be sending another pull request with the updates the fix the issue in a little bit...

File: libavogadro/src/extensions/molecularpropextension.cpp [hunk 1]
```diff
@@ -207,8 +210,23 @@ namespace Avogadro {
       m_molecule->setProperty("name", QVariant()); // set an invalid name, since we don't have one
   }
 
+  void MolecularPropertiesExtension::printSslErrors(
+                                                QNetworkReply*, 
+                                                const QList<QSslError> &errors)
+  {
+    foreach(const QSslError &error, errors) {
+      qDebug() << QString("SSL Error: %1").arg(error.errorString());
```
Comment (ghutchis) [inline libavogadro/src/extensions/molecularpropextension.cpp]: Even though this is debugging, I think we'd want to use tr("SSL Error %1") to allow localization.

File: libavogadro/src/extensions/molecularpropextension.cpp [hunk 2]
```diff
@@ -207,8 +210,23 @@ namespace Avogadro {
...
+    }
+  }
+
   void MolecularPropertiesExtension::replyFinished(QNetworkReply *reply)
   {
+    // Print error messages
+    if (reply->error() != QNetworkReply::NoError) {
+      qDebug() << QString("Network Error: %1").arg(reply->errorString());
```
Comment (ghutchis) [inline libavogadro/src/extensions/molecularpropextension.cpp]: Ditto tr()

File: libavogadro/src/extensions/networkfetchextension.cpp [hunk 3]
```diff
@@ -184,8 +187,22 @@ namespace Avogadro
     return redirectUrl;
   }
 
+  void NetworkFetchExtension::printSslErrors(QNetworkReply*, 
+                                             const QList<QSslError> &errors)
+  {
+    foreach(const QSslError &error, errors) {
+      qDebug() << QString("SSL Error: %1").arg(error.errorString());
```
Comment (ghutchis) [inline libavogadro/src/extensions/networkfetchextension.cpp]: tr()

File: libavogadro/src/extensions/networkfetchextension.cpp [hunk 4]
```diff
@@ -184,8 +187,22 @@ namespace Avogadro
...
+    }
+  }
+
   void NetworkFetchExtension::replyFinished(QNetworkReply *reply)
   {
+    // Print error messages
+    if (reply->error() != QNetworkReply::NoError) {
+      qDebug() << QString("Network Error: %1").arg(reply->errorString());
```
Comment (ghutchis) [inline libavogadro/src/extensions/networkfetchextension.cpp]: tr() .. might also want to add a few other possible network error cases (e.g., server is down) to help with debugging.

**Comment by psavery** [comment]
Okay. I updated them to use tr(). 

It does print a variety of network errors depending on why it couldn't connect. Qt sets the error  We could potentially add more cases if more came up.

**Pull request merged**